gcloud/pubsub/client.py

```diff
@@ -50,6 +50,12 @@ def list_topics(self, page_size=None, page_token=None):
         See:
         https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/list
 
+        Example:
+
+        .. literalinclude:: pubsub_snippets.py
+           :start-after: [START client_list_topics]
+           :end-before: [END client_list_topics]
```

Comment:
> What evidence do we have that printing examples at the Python REPL prompt is important

I personally use `help(foo)` or `foo?` (in IPython) quite a lot. Though usually it's for signatures, not for examples. (Also that isn't "evidence".)

> the only way to ensure that the don't bitrot.

Saying *only* is a bit much. However, I'm very happy with examples pulled from an actual Python file.